code/style.css

```diff
@@ -1,4 +1,229 @@
+    text-align: center;
+    margin-bottom: -10px;
+    }
+
+    .tagline {
+    text-align: left;
+    text-decoration-line: underline;
+    font-size: 20px;
+    color: red;
+    margin-top: 1%;
+    }
+
+  /* header items */
+
+   .navbar {
+    padding-top: 5%;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    visibility: hidden;
+    }
+
+   .logo {
+    height: 100px;
+    padding: 50px;
+    margin-bottom: 20px;
+    margin-left: 25px;
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+    }
+
+    .logo img {
+     height: 100%;
+     }
+
+    .container {
+     display: flex;
+     justify-content: center;
+     align-items: center;
+     }
+
+    .big-news {
+    background-color: seashell;
+    width: 60%;
+    padding: 2% 5% 2% 5%;
+    display: flex;
+    flex-direction: column;
+    }
+
+    .big-news img {
+    width: 100%;
+    }
+
+    .news-grid {
+     width: 60%;
+     padding: 2% 5% 2% 5%;
+     margin-top: 5%;
+     display: flex;
+     justify-content: center;
+     align-items: center;
+     flex-direction: column;
+     flex-wrap: nowrap;
+     }
+
+    .news-card {
+     background-color: seashell;
+     height: auto;
+     width: 100%;
+     padding: 5%;
+     margin-top: 5%;
+     display:flex;
+     flex-direction: column;
+     }
+
+    .news-card img {
+     width: 100%;
+     }
+
+ /* tablet */
+
+@media (min-width: 668px) and (max-width: 1024px) {
+    
+    body {
+    background-color: bisque;
+    }
+
+    .navbar {
+    visibility: visible;
+    }
+
+    .logo {
+    height: 150px;
+    margin-top: 2%;
+    margin-bottom: 5%;
+    justify-content: center;
+    align-items: center;
+    }
+   
+   .big-news img {
+    margin-bottom: -40px;
+    }
+
+    h4 {
+    font-size: 20px;
+    }
+
+    h3 {
+    font-size: 20px;
+    }
+
+    p {
+    margin-top: 10%;
+    font-size: 15px;
+    }
+
+    .top-story {
+    font-size: 30px;
+    }
+
+
+    .news-grid {
+    width: 70%;
+    margin-top: 20px;
+    flex-direction: row; 
+    flex-wrap: wrap;
+    }
+
+    .news-card {
+    height: 300px;
+    width: 300px;
+    margin: auto;
+    margin-top: 20px;
+    flex-direction: column;
+    flex-basis: 38%;
+    }
+
 }
+
+/* desktop*/
+
+@media (min-width: 1025px) {
+  
+   body {
+    background-color: pink;
+    }
+
+   .navbar {
+    visibility: visible;
+    }
+
+   .logo {
+   height: 200px;
+   margin-top: 2%;
+   margin-bottom: 5%;
+   justify-content: center;
+   align-items: center;
+   }
+
+   h4 {
+   font-size: 20px;
+   }
+
+   h3 {
+   font-size: 30px;
+   }
+
+   p {
+   margin-top: 10%;
+   font-size: 18px;
+   }
+
+   .top-story {
+   font-size: 50px;
+   }
+
+   .big-news img {
+   margin-bottom: -40px;
+   }
+
+   .news-grid {
+   width: 70%;
+   flex-direction: row;
+   }
+
+   .news-card {
+   height: 200px;
+   width: 200px;
+   margin: auto;
+   margin-top: 20px;
+   flex-direction: column;
+   flex-basis: 14%;
+   }
+
+}
```

Comment:
SUMMARIZING COMMENTS:

Overall, super impressed by your site - I do remember Friday you told us you hadn't begun yet, and you have fulfilled all the minimum requirements laid out in the brief.

Some things that I noticed in the site, but which I wasn't sure where to comment:
There was some inconsistency using margins and paddings, both in values, but also a good mix of % and pixels. I can see why - you were trying to align parts your site. I think in the future, it would be easier sticking to one or the other. Another tip is to use the same left-right padding on the section containers - that way you don't have to add it to the actual elements inside the site, and it makes it easier to see what's going on, and to keep things aligned.

HTML also automatically adds padding to things like h1, h2, h3, h4, p, etc - in the beginning of the CSS document, it can be useful to remove all default padding by setting it to 0, so you can control it better yourself in the rest of the document.

I can recommend a Chrome extension called "Pesticide" - when you activate it, you can see all the boxes and margins/padding of how the site is built up. It makes it a lot easier to inspect when using the inspect tool in Chrome and see what's going on "behind the scenes", since all our sites are made of boxes.

Overall I think you used flex really well, and your content was a fun read! Awesome job!

❤️ Michael